tf_seal/cc/kernels/seal_kernels.cc

```diff
@@ -79,6 +79,58 @@ Status LookupOrCreateWrapper(OpKernelContext* ctx,
                                                      });
 }
 
+class SealSavePbOp : public OpKernel {
+ public:
+  explicit SealSavePbOp(OpKernelConstruction* ctx): OpKernel(ctx) {}
+
+
+  void Compute(OpKernelContext* ctx) override {
+      const PublicKeysVariant* key = nullptr;
+    OP_REQUIRES_OK(ctx, GetVariant(ctx, 1, &key));
+      Tensor* out0;
+    OP_REQUIRES_OK(ctx, ctx->allocate_output(0, TensorShape{}, &out0));
+    seal::PublicKey publicKey(key->public_key);
+    std::filebuf fb;
+    fb.open("public_key", std::ios::out);
+    std::ostream pubk(&fb);
+    publicKey.save(pubk);
+  }
+};
+
+
+class SealSaveScOp : public OpKernel {
+ public:
+  explicit SealSaveScOp(OpKernelConstruction* ctx): OpKernel(ctx) {}
+
+  void Compute(OpKernelContext* ctx) override {
+     const SecretKeyVariant* secretkey = nullptr;
+    OP_REQUIRES_OK(ctx, GetVariant(ctx, 1, &secretkey));
+    Tensor* out0;
+    OP_REQUIRES_OK(ctx, ctx->allocate_output(0, TensorShape{}, &out0));
+    seal::SecretKey secretKey(secretkey->key);
+    std::filebuf fb;
+    fb.open("secret_key", std::ios::out);
+    std::ostream pubk(&fb);
+    secretKey.save(pubk);
```

Comment:
it's probably a good idea to explicitly close the file when we're done with it; ideally adding a bit of error handling as well

Reply:
Done in next commit